### PR TITLE
feat/#58

### PR DESCRIPTION
**Pull request by locxuandao**

### CODE REVIEW

File: src/app/components/ButtonCustomization/ButtonCustomization.module.scss [hunk 1]
```diff
@@ -24,3 +24,6 @@
     transform: rotate(360deg);
   }
 }
```
Comment (nc-minh) [inline src/app/components/ButtonCustomization/ButtonCustomization.module.scss:26]: There should be a space here

File: src/app/components/ButtonCustomization/index.tsx [hunk 2]
```diff
@@ -11,14 +12,19 @@ interface Props {
...
   className?: string;
   isLoading?: boolean;
+  page?: string;
+  prefix?: string;
+  id?: string;
 }
 
 export const ButtonCustomization = (props: Props) => {
-  const { children, onClick, className, isLoading } = props;
+  const { children, onClick, className, isLoading, page, prefix, id } = props;
```
Comment (nc-minh) [inline src/app/components/ButtonCustomization/index.tsx:21]: What is this? Why Link wrap on the button, these 2 have different components. Not allowed to share, violate the rules **Single responsibility principle**, where the button wraps a button, not a link.

File: src/app/containers/TestViewWrapper/index.tsx [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+          <Typography className={cx("highlight")} component="strong">
+            {questionsByExamId?.exam?.subject?.subject_name}
+          </Typography>
+        </div>
+
+        <div className={cx("examItem")}>
+          <Typography className={cx("nameItem")} component="p">
+            Tác giả đăng
+          </Typography>
+          <Typography className={cx("colon2")} component="p">
```
Comment (nc-minh) [inline src/app/containers/TestViewWrapper/index.tsx:45]: Colon2?

**Review by nc-minh — CHANGES_REQUESTED** [review]

File: src/app/components/ButtonCustomization/index.tsx (hunk 2 above)
Comment (locxuandao) [reply src/app/components/ButtonCustomization/index.tsx:21]: well,from now on i know

File: src/app/containers/TestViewWrapper/index.tsx (hunk 3 above)
Comment (locxuandao) [reply src/app/containers/TestViewWrapper/index.tsx:45]: there's a difference between the colons so I've given them different classnames :  colon1 and colon2. That shouldn't matter much, will it?

File: src/app/components/ButtonCustomization/index.tsx (hunk 2 above)
Comment (locxuandao) [reply src/app/components/ButtonCustomization/index.tsx:21]: 339827453_208882511784271_3529495507787710818_n

Comment (locxuandao) [reply src/app/components/ButtonCustomization/index.tsx:21]: I learned about SRP and now I have fixed it like this, is that correct?

Comment (nc-minh) [reply src/app/components/ButtonCustomization/index.tsx:21]: okay cool

File: src/app/containers/TestViewWrapper/index.tsx (hunk 3 above)
Comment (nc-minh) [reply src/app/containers/TestViewWrapper/index.tsx:45]: oke

**Review by nc-minh — APPROVED** [review]

**Pull request merged**